src/makeRobotAccountant.js

```diff
@@ -6,7 +6,23 @@
  */
 
 function makeRobotAccountant() {
-  // write code here
+  let count = 0;
+
+  const getSum = (num1 = 0) => {
```

Comment:
Instead of defining the functions and then returning them, you can return them directly here. Because these functions aren't used anywhere else in the makeRobotFuntion.

Reply:
good point from @nurdanokcu, let's do this task this way